tft/data_formatters/volatility.py

```diff
@@ -180,7 +180,7 @@ def format_predictions(self, predictions):
 
     for col in column_names:
       if col not in {'forecast_time', 'identifier'}:
-        output[col] = self._target_scaler.inverse_transform(predictions[col])
+        output[col] = self._target_scaler.inverse_transform(predictions[col].values.reshape(-1,1))
```

Comment:
Have you tested this trains without error?

Reply:
Yes, of cause.